bindings/guile/gnc-engine-guile.cpp

```diff
@@ -1909,3 +1909,13 @@ gnc_parse_time_to_time64 (const gchar *s, const gchar *format)
     return gnc_mktime(&tm);
 }
 
+
+gnc_numeric
+gnc_string_to_number (const char * in_str)
+{
+    gnc_numeric rv;
+    if (!string_to_gnc_numeric (in_str, &rv))
+        rv = gnc_numeric_create (1, 0);
+    return rv;
+}
```

Comment:
Adding a `%typemap(argout) gnc_numeric` to common/base-typemaps.i would allow `string_to_gnc_numeric` to be wrapped correctly and make this unnecessary.

Reply:
Alternatively `string_to_gnc_numeric` is used in only 6 places. You could rewrite it to `gnc_numeric gnc_numeric_from_string(const char*)` returning `GNC_ERROR_ARG` on failure and change those 6 uses.